book/website/collaboration/research-infrastructure-roles.md

```diff
@@ -32,3 +30,21 @@ But this work is often hidden and not visible in academia, where papers and cita
 The motivation for writing this chapter and providing examples of the type of work that goes into research infrastructure is to make this hidden labour visible to a wider audience and highlight just how critical these aspects of research are for any community or project. 
 
 Other initiatives such as the [Software Sustainability Institute](https://www.software.ac.uk/), the [Center for Scientific Collaboration & Community Engagement](https://www.cscce.org/), and the [Hidden REF](https://hidden-ref.org/), are all contributing to the growing recognition of the importance of these roles for research. 
+
+(cl-research-infrastructure-roles-list)=
+## Example Role Title List
```

Comment:
@Arielle-Bennett Thoughts on renaming this to:

"List of Research Infrastructure Roles" or "Examples of Research Infrastructure Role"

The existing title in the header is somewhat confusing to me